Update changelog with company.country related changes (#50)
BEEQUIP/developer.beequip.nl: src/pages/marketplace/changelog.mdx

## Past changes

### 30 September 2025

- Field `country` was added to type `Company`.

> - Mutations `AddCompany` and `UpdateCompany` accept the optional `country` input field.

Reviewer: I would like to keep this in line with the other entries in the changelog, when other fields were added.

```suggestion
- Field `country` was added to the types `AddCompanyInput` and `UpdateCompanyInput`.
```
